kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -451,6 +455,30 @@ interface ViewPagerAssertions : BaseAssertions {
 }
 
 /**
+ * Provides assertios for RatingBar
+ */
+interface RatingBarAssertions : BaseAssertions {
+    /**
+     *  Checks If rating bar is have number of rating as expected
+     *  @param number rating as expected
+     */
+    fun expectedNumberOfStarAs(number: Float) {
+        view.check(ViewAssertions.matches(object : TypeSafeMatcher<View>() {
+            override fun matchesSafely(item: View): Boolean {
+                if (RatingBar::class.java.isAssignableFrom(item.javaClass)) {
+                    val ratingBar = item as RatingBar
+                    return ratingBar.rating == number
+                } else return false
+            }
+
+            override fun describeTo(description: Description) {
+                description.appendText("star rating is: " + number)
```

Comment:
Rating bar does not appears as stars in all cases. I believe it is safe to output `"rating value is: $number"`